contrib/IECoreAlembic/src/IECoreAlembic/CameraReader.cpp

```diff
@@ -87,12 +87,18 @@ class CameraReader : public IECoreAlembic::ObjectReader
 			cameraSchema.get( sample, sampleSelector );
 
 			CameraPtr result = new Camera;
-			result->parameters()["projection"] = new StringData( "perspective" );
-
-			double top, bottom, left, right;
-			sample.getScreenWindow( top, bottom, left, right );
-			result->parameters()["screenWindow"] = new Box2fData( Box2f( V2f( left, bottom ), V2f( right, top ) ) );
-			result->parameters()["projection:fov"] = new FloatData( sample.getFieldOfView() );
+			result->setProjection( "perspective" );
+
+			result->setFocalLength( sample.getFocalLength() );
+			result->setAperture( 10.0f * Imath::V2f(
+				sample.getHorizontalAperture(), sample.getVerticalAperture() ) );
+			result->setApertureOffset( 10.0f * Imath::V2f(
+				sample.getHorizontalFilmOffset(), sample.getVerticalFilmOffset() ) );
+
+			result->setClippingPlanes( Imath::V2f( sample.getNearClippingPlane(),sample.getFarClippingPlane() ) );
+			result->setFStop( sample.getFStop() );
+			result->setFocalLengthWorldScale( 0.1 ); // Alembic stores focal length in tenths of world units
```

Comment:
Is this definite? I know they document focal length as being mm, but I'm not aware of documentation that world units are cm.

Reply:
Yep.

https://github.com/alembic/alembic/blob/master/lib/Alembic/AbcGeom/CameraSample.h#L145

I guess technically they don't explicitly state that focusDistance should be in world units, but they do require all the things that you would expect to be in world units to be in centimeters.

Reply:
I'd seen that, but was sceptical that it had actually been applied consistently throughout the Alembic toolchain. But I just made a Maya scene in inches and exported a unit-radius sphere, and lo and behold it has a radius of 2.54 in Alembic/Gaffer.